### DISCUSSION TITLE
Text near bot

### DISCUSSION THREAD

**Discussion by jeffh86** [FlowiseAI/Flowise#1453] [Ideas]
![example](https://github.com/FlowiseAI/Flowise/assets/142155913/07cb3dd3-be8e-4228-8d2a-36aead4ab2d3)

Lots of chatbots I have seen have little messages by them to increase the chances of interactions. Is there any chance that this feature could be added?

Thanks for the amazing work by the way!

**Comment by HenryHengZJ** [comment]
@0xi4o

**Reply by 0xi4o** [reply]
This would be a nice addition and should be trivial to implement — a `popupMessage` option in the configuration. 

When should the popup message be shown? Chatbots usually wait for a few seconds before showing the messages. Should the wait time be configurable too?

**Reply by jeffh86** [reply]
Not that the development has anything to do with me 😊, but the fade-in animation of the current chatbot looks great. I think if it appeared at the same time it would look cool.

**Comment by dimando** [comment]
Thank you for this idea, would be very useful.
Also would be fantastic to have a form to collect some customer details (like name, email, phone, etc).

**Comment by amansoni7477030** [comment]
Hi @jeffh86, I recently added this feature in https://github.com/FlowiseAI/FlowiseChatEmbed/pull/192. This pull request is awaiting approval.

**Comment by simosme** [comment]
Thank you for the implementation, @amansoni7477030 - currently I can't find any way to allow users to close the tooltip, or to make it disappear after some time. Is this possible or not a feature yet?

**Reply by amansoni7477030** [reply]
> Thank you for the implementation, @amansoni7477030 - currently I can't find any way to allow users to close the tooltip, or to make it disappear after some time. Is this possible or not a feature yet?

@simosme, actually that's not a feature currently, but it might be added in the future.
